data/map.txt

```diff
@@ -30694,3 +30700,2835 @@ planet Zug
+			distance 487
+			period 334.25
+	object
+		sprite planet/jupiter
+		distance 8125
+		period 31842.5
+
+system 31b
+	pos 309.8 -10.74
+	government Uninhabited
+	habitable 28275.2
+	belt 1795
+	haze _menu/haze-full
+	link 22b
+	link 23b
+	link 44b
+	asteroids "small rock" 25 0.67
+	asteroids "medium rock" 54 0.84
+	asteroids "large rock" 15 0.61
+	asteroids "small metal" 5 0.64
+	asteroids "medium metal" 3 1.04
+	asteroids "large metal" 4 0.99
+	minables aluminum 25 0.95
+	object
+		sprite star/m-giant
+		distance 1.48
+		period 10
+	object
+		sprite planet/cloud0
+		distance 4567.34
+		period 151.58
+
+system 34b
+	pos 401.41 -61.62
+	government Uninhabited
+	habitable 6195.2
+	belt 1944
+	haze _menu/haze-fullbright
+	link 21b
+	asteroids "large metal" 1 2.2
+	object
+		sprite star/g-giant
+		period 10
+	object
+		sprite planet/neptune-b
+		distance 578
+		period 157.9
+	object
+		sprite planet/venus
+		distance 1263
+		period 510.06
+	object
+		sprite planet/gas15-b
+		distance 2253
+		period 1215.2
+	object
+		sprite planet/rock0
+		distance 2757
+		period 1645.02
+
+planet "Giaru Gegno"
+	attributes "gegno quarg" quarg station
+	landscape land/station2
+	description "The quality of its structure suggests that this Quarg Ringworld had to have been completed rather recently, but there are some notable differences that don't seem Quarg-like. Some of the hangers and docks on the ring are oddly larger than others, and the inner quarters are wider and taller than what is normal for a Quarg. The hangar you currently are parked in, however, seems to be void of these characteristics, and is similar to ones at the Quarg ringworld in human space."
+	description `	Despite seeing numerous alien craft before landing on the ring, you are only guided around parts of the ring where they are not present.` 
+	spaceport "There has to be at least billions of Quarg on this ring. They shift past you at a normal walking pace for a Quarg, and don't give you much attention except from the occasional uninterested glance or two. There seems to be little conversation, and the silence is slightly unnerving to you as the only non-Quarg in sight. It seems areas containing the aliens from nearby are off-limits to you."
+	government Quarg
+	bribe 0
+	security 1
+
+planet Ternituul
+	landscape land/fog1
+	description "Ternituul is a dull world covered in a thick layer of fog hiding a dry desolate surface; an ideal world for the hardened Gegno Vi. A large portion of the Vi's military branch is based here, where new recruits are thrown into the first bottleneck of their lives. It is not uncommon for inhabitants to be lost to the bleak ecosystem of this planet."
+	description "	The fog also hides a more grim aspect of this world- numerous graveyards cover the terrain, serving as a final resting place for most Vi. Large ceremonies are constantly held for the fallen, though instead of grievance, the Vi celebrate the commitment and dignity of their fellow brethren."
```

Comment:
```suggestion
	description "	The fog also hides a more grim aspect of this world - numerous graveyards cover the terrain, serving as a final resting place for most Vi. Large ceremonies are constantly held for the fallen, though instead of grievance, the Vi celebrate the commitment and dignity of their fellow brethren."
```